openpype/hosts/maya/plugins/load/load_yeti_rig.py

```diff
@@ -22,72 +19,31 @@ class YetiRigLoader(openpype.hosts.maya.api.plugin.ReferenceLoader):
-
-        import maya.cmds as cmds
-
-        # get roots of selected hierarchies
-        selected_roots = []
-        for sel in cmds.ls(sl=True, long=True):
-            selected_roots.append(sel.split("|")[1])
-
-        # get all objects under those roots
-        selected_hierarchy = []
-        for root in selected_roots:
-            selected_hierarchy.append(cmds.listRelatives(
-                                      root,
-                                      allDescendents=True) or [])
-
-        # flatten the list and filter only shapes
-        shapes_flat = []
-        for root in selected_hierarchy:
-            shapes = cmds.ls(root, long=True, type="mesh") or []
-            for shape in shapes:
-                shapes_flat.append(shape)
-
-        # create dictionary of cbId and shape nodes
-        scene_lookup = defaultdict(list)
-        for node in shapes_flat:
-            cb_id = lib.get_id(node)
-            scene_lookup[cb_id] = node
+        self, context, name=None, namespace=None, options=None
+    ):
 
-        # load rig
+        group_name = "{}:{}".format(namespace, name)
         with lib.maintained_selection():
-            file_url = self.prepare_root_value(self.fname,
-                                               context["project"]["name"])
-            nodes = cmds.file(file_url,
-                              namespace=namespace,
-                              reference=True,
-                              returnNewNodes=True,
-                              groupReference=True,
-                              groupName="{}:{}".format(namespace, name))
-
-        # for every shape node we've just loaded find matching shape by its
-        # cbId in selection. If found outMesh of scene shape will connect to
-        # inMesh of loaded shape.
-        for destination_node in nodes:
-            source_node = scene_lookup[lib.get_id(destination_node)]
-            if source_node:
-                self.log.info("found: {}".format(source_node))
-                self.log.info(
-                    "creating connection to {}".format(destination_node))
-
-                cmds.connectAttr("{}.outMesh".format(source_node),
-                                 "{}.inMesh".format(destination_node),
-                                 force=True)
-
-        groupName = "{}:{}".format(namespace, name)
-
-        settings = get_project_settings(os.environ['AVALON_PROJECT'])
-        colors = settings['maya']['load']['colors']
+            file_url = self.prepare_root_value(
+                self.fname, context["project"]["name"]
+            )
+            nodes = cmds.file(
+                file_url,
+                namespace=namespace,
+                reference=True,
+                returnNewNodes=True,
+                groupReference=True,
+                groupName=group_name
+            )
 
-        c = colors.get('yetiRig')
+        settings = get_project_settings(os.environ["AVALON_PROJECT"])
```

Comment:
Should we replace this with 
```python
from openpype.settings import get_current_project_settings
settings = get_current_project_settings()
```
To rely less on `os.environ['AVALON_PROJECT']` directly in the code?
I believe it's a goal to clean that up, right @iLLiCiTiT ?

Reply:
Done